content/docs/about/policies/listeneroption.md

```diff
@@ -96,12 +96,12 @@ spec:
 
 ## Conflicting policies
 
-If you create multiple ListenerOption resources and attach them to the same gateway listener by using the `targetRefs` option, only the ListenerOption that was first created is applied. 
+If you create multiple ListenerPolicy resources and attach them to the same gateway listener by using the `targetRefs` option, only the ListenerPolicy that was first created is applied. 
```

Comment:
```suggestion
If you create multiple ListenerPolicy resources and attach them to the same gateway listener by using the `targetRef` option, only the ListenerPolicy that was first created is applied. 
```

Reply:
should this say ... by creating multiple ListenerPolicy resources that are applied to the same gateway listener?